src/seedu/addressbook/data/tag/Tagging.java

```diff
@@ -0,0 +1,23 @@
+package seedu.addressbook.data.tag;
+
+import seedu.addressbook.data.person.Person;
+
+/**
+ * Represents an addition or deletion of a tag in the address book
+ */
+public class Tagging {
+	private Person personInvolved;
+	private Tag tagInvolved;
+	private String action;
```

Comment:
Not a good idea to maintain the action as a String type. Consider enum or boolean